### PR TITLE
[TE] migrate logging from logback to log4j2

### PR DESCRIPTION
**Pull request by vincentchenjl**

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/RCAFrameworkRunner.java [hunk 1]
```diff
@@ -124,23 +115,6 @@ public static void main(String[] args) throws Exception {
...
-    if (cmd.hasOption(CLI_LOG_DEBUG))
-      setLogger(cmd.getOptionValue(CLI_LOG_DEBUG), Level.DEBUG);
-    if (cmd.hasOption(CLI_LOG_INFO))
-      setLogger(cmd.getOptionValue(CLI_LOG_INFO), Level.INFO);
-    if (cmd.hasOption(CLI_LOG_WARN))
-      setLogger(cmd.getOptionValue(CLI_LOG_WARN), Level.WARN);
-    if (cmd.hasOption(CLI_LOG_ERROR))
-      setLogger(cmd.getOptionValue(CLI_LOG_ERROR), Level.ERROR);
-
-    if(cmd.hasOption(CLI_VERBOSE))
```
Comment (akshayrai) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/RCAFrameworkRunner.java:141]: Can't we set the log level in log4j2?

Comment (vincentchenjl) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/RCAFrameworkRunner.java:141]: We are using `slf4j` for logger across the codebase and there is no way to set the log level in runtime with `slf4j`. Also, this class is standalone and actually never uses any logger inside. If we want to set the log level, we can always specify log4j2 config when run this class, which is common practice for log4j2.

Comment (akshayrai) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/RCAFrameworkRunner.java:141]: Thanks. Got it. Can you remove the variable declarations as well?
```private static final String CLI_LOG = "log";
  private static final String CLI_LOG_DEBUG = "log-debug";
  private static final String CLI_LOG_INFO = "log-info";
  private static final String CLI_LOG_WARN = "log-warn";
  private static final String CLI_LOG_ERROR = "log-error";```

Comment (vincentchenjl) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/RCAFrameworkRunner.java:141]: Done.

**Pull request merged**